app/templates/components/course-rollover.hbs

```diff
@@ -30,7 +30,14 @@
     {{#liquid-if expandAdvancedOptions class='vertical'}}
       <div class='item'>
         <label>{{t "general.startDate"}}:</label>
-        {{pikaday-input size=10 value=startDate maxDate=maxDate minDate=minDate format='YYYY-MM-DD'}}
+        {{pikaday-input
+          size=10
+          value=startDate
+          maxDate=maxDate
+          minDate=minDate
+          format='YYYY-MM-DD'
+          onSelection=(action 'selectStartDate')
```

Comment:
Alternatively this can be done without creating an action method on the component with 
`onSelection=(action mut startDate)`

Its valid either way.